bot-github-core/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Using a parent for dependency versions,
+       code formatting, and plugins. -->
+    <parent>
+        <groupId>org.commonhaus.automation</groupId>
+        <artifactId>bot-parent</artifactId>
+        <version>${revision}</version>
+        <relativePath>../pom.xml</relativePath>
```

Comment:
This is the default value. Do you have to set it explicitly for a reason?

Reply:
I seem to remember things being weird without it... at some point. ;)